src/coreclr/jit/instr.h

```diff
@@ -47,6 +51,14 @@ enum instruction : unsigned
 
     INS_lea,   // Not a real instruction. It is used for load the address of stack locals
 
+#elif defined(TARGET_LOONGARCH64)
+    #define INSTS(id, nm, fp, ldst, fmt, e1) INS_##id,
```

Comment:
This is a minor "nit": I would name the define `INST` instead of `INSTS`. That is, not trailing "S". That trailing "S" in English makes it seem like it is plural, but it isn't: each `INST` defines one single instruction.

Reply:
OK, Thanks